parse_emails/handle_eml.py

```diff
@@ -273,6 +276,27 @@ def handle_eml(file_path, b64=False, file_name=None, parse_only_headers=False, m
         return email_data, attached_emails
 
 
+def handle_multi_part_error(eml):
+    """
+    This function handles a multipart which is missing boundary,
+    and checks if the boundary exists only ons in the file, it removes that.
+    Args:
+        the eml parse obj (Message) : the email data.
+    Returns:
+        the eml parse obj (Message)
+    """
+    for defect in eml.defects:
+        if isinstance(defect, errors.MultipartInvariantViolationDefect):
+            boundary = eml.get_boundary()
+            file_data = eml.as_string()
+            if file_data.count(boundary) == 1:
+                param = eml.get("Content-Type").replace('\r\n', '\n')
+                file_data = eml.as_string().replace(f'Content-Type: {param}\n', '')
+                eml = message_from_string(file_data)
+            break
```

Comment:
why break? it can't be more than once?

Reply:
@moishce what about this?

Reply:
It just goes through the list of errors returned, and we check if the error we are looking for is there
if we find it, we can stop